index.d.ts

```diff
@@ -46,20 +46,20 @@ import { FastImageProps } from "react-native-fast-image";
 import { theme as themeDef } from "./src/theme";
 import { BUTTON_VARIANTS } from "./src/components/Button";
 
-type addPrefixToObject<
-  TObject extends object,
+type addPrefixToo<
+  To extends o,
```

Comment:
@kiran-kumar011 Looks like `Object` got replaced by `o` in many places, Please check.

Reply:
I have cross verified with the previous changes and replaced it in all the places.